.github/workflows/ci.yaml

```diff
@@ -143,14 +143,18 @@ jobs:
         id: vscode-rev
         run: echo "::set-output name=rev::$(git rev-parse HEAD:./lib/vscode)"
 
-        # We need to rebuild when we have a new version of Code or when any of
-        # the patches changed.  Use VSCODE_CACHE_VERSION to force a rebuild.
+      - name: Get version
+        id: version
+        run: echo "::set-output name=version::$(jq -r .version package.json)"
+
+      # We need to rebuild when we have a new version of Code or when any of
+      # the patches changed.  Use VSCODE_CACHE_VERSION to force a rebuild.
```

Comment:
```suggestion
      # We need to rebuild when we have a new version of Code, when any of
      # the patches changed, or when the code-server version changes (since 
      # it gets embedded into the code).  Use VSCODE_CACHE_VERSION to 
      # force a rebuild.
```